### PR TITLE
Added colours

### PR DESCRIPTION
**Pull request by A-Manning**
Uses the golden ratio reciprocal, a random seed,  and a HSV colour space to generate random, distinct, and bright colours

### CODE REVIEW

**Comment by lexcast** [comment]
I really think this should be optional, by a setting to select use the actual behavior or this random color.

**Comment by A-Manning** [comment]
Absolutely agree. I couldn't figure out how to make something optional though. Any chance I could have some help?

**Comment by lexcast** [comment]
https://atom.io/docs/api/v1.10.2/Config you could use a enum type setting, letting the user select between the cursor position color (default) or the random color.

**Comment by A-Manning** [comment]
@lexcast @JoelBesada  colours are now optional, with behaviour unchanged as default.

File: lib/config-schema.coffee [hunk 1]
```diff
@@ -38,6 +38,11 @@ module.exports =
         description: "Turn the particles on/off."
         type: "boolean"
         default: true
+      colours:
+        title: "Colours - Enabled"
+        description: "Turn colours on/off"
+        type: "boolean"
+        default: false
```
Comment (lexcast) [inline lib/config-schema.coffee]: this should be an enum type, given I want to add another option later for a fixed color.

Comment (A-Manning) [inline lib/config-schema.coffee]: On it. Will be done today or tomorrow

Comment (lexcast) [inline lib/config-schema.coffee]: Great, Thank you

Comment (A-Manning) [inline lib/config-schema.coffee]: Ok, colour options are now enumerable.

**Comment by lexcast** [comment]
thank you :tada:

**Pull request merged**